components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -3449,15 +3449,15 @@ subroutine ocn_compute_land_ice_flux_input_fields(layerThickness, normalVelocity
 
       ! modify the spatially-varying attenuation coefficient where there is land ice
 #ifdef MPAS_OPENACC
-      !$acc enter data copyin(landIceMask)
+      !$acc enter data copyin(landIceFloatingMask)
 
-      !$acc parallel loop present(landIceMask, sfcFlxAttCoeff)
+      !$acc parallel loop present(landIceFloatingMask, sfcFlxAttCoeff)
 #else
       !$omp parallel
       !$omp do schedule(runtime)
 #endif
       do iCell = 1, nCellsAll
-         if(landIceMask(iCell) == 1) then
+         if(landIceFloatingMask(iCell) == 1) then
             sfcFlxAttCoeff(iCell) = config_land_ice_flux_attenuation_coefficient
```

Comment:
We presumably don't care what this value is in the thin film region, since there are no surface fluxes?

Reply:
I'm not super familiar with this forcing option, but I think we might want to keep this, in case at some point we do want to use surface forcing fields in ice shelf cases (I think you mean ice shelf cavities, not the thin film region). Otherwise, we should probably remove `config_land_ice_flux_attenuation_coefficient` and completely disable this option.

Reply:
We do use surface fluxes in the ice-shelf cavity so we need to keep his option.  I think I would have left this as `landIceMask` since it makes more sense to me that the attenuation coefficient is the same in the thin film as in the ice-shelf cavity but I think it doesn't matter what the thin film values is because we don't allow surface fluxes there.  So whether this is left as you have it or reverted back to what it used to be would make no material difference.  That was my point.

Reply:
Oh, I see what you mean now. Yeah, it definitely wouldn't be used in the thin film so I thought it would be more clear to have it assigned only where it would be used.

Reply:
That's fine.  The way you have it here, the attenuation coefficient (the length scale over which surface fluxes are exponentially distributed) will take on the open ocean value in the thin film region.  So it's not a field that is "off" on the open ocean and only on in cavities or anything like that.  But like I said it doesn't matter which value it has in the thin film region so we can leave your change.